src/lib/colorUtils.ts

```diff
@@ -0,0 +1,50 @@
+import {ColorRgb, Option} from "@/types/common.types";
+
+const RED = 0.2126;
+const GREEN = 0.7152
+const BLUE = 0.0722;
+const GAMMA = 2.4;
```

Comment:
what are these values exactly?, maybe a more indicative name instead of "RED" etc

Reply:
https://stackoverflow.com/questions/9733288/how-to-programmatically-calculate-the-contrast-ratio-between-two-colors